exercises/proverb/canonical-data.json

```diff
@@ -0,0 +1,98 @@
+{
+    "exercise": "proverb",
+    "version": "1.0.0",
+    "comments": [
+       "JSON doesn't allow for multi-line strings, so all expected outputs are presented ",
+       "here as arrays of strings. It's up to the test generator to join the ",
+       "lines together with line breaks."
+    ],
+    "cases": [
+        {
+            "cases": [
+                {
+                    "description": "zero pieces",
+                    "expected": [
```

Comment:
I was considering to ask that this should have zero elements, but given that this is converted to a string by joining all elements with newlines, then `[""]` and `[]` are both representations of the empty string, so do whatever you want.